docs/docs/examples/bottom-sheet.mdx

```diff
@@ -0,0 +1,10 @@
+---
+id: BottomSheet
+sidebar_label: Bottom Sheet
+---
+
+
```

Comment:
there definitely have to be some short description and some videos before every example